RHDEVDOCS-6341: Content creation for the NamespaceManagement CR (#98500)
openshift/openshift-docs: modules/gitops-enable-tenant-namespace-management-with-namespacemanagement.adoc

// Module included in the following assemblies:
//
// * multitenancy/multitenancy-support-in-gitops.adoc

:_mod-docs-content-type: CONCEPT
[id="gitops-enable-tenant-namespace-management-with-namespacemanagement_{context}"]
= Enable tenant namespace management with the NamespaceManagement CR

By default, only a cluster administrator can enable Argo CD to manage a namespace. This is done by labeling the namespace with `argocd.argoproj.io/managed-by: <argo cd namespace>`. This model requires `cluster-admin` privileges and is not suitable for tenant self-service.

> .How does this `NamespaceManagement` CR work?

Reviewer: block titles are not supported in dita. I suggest you to think of an alternative (everywhere in the doc). 
Suggestions:

-  _How does this `NamespaceManagement` CR work?_ instead of .  (to have italics)
- replace this with the
- Can we have title as 'Working of `NamespaceManagement` CR instead of a question?

Are we allowed to use qs as a title?

Author: Fixed. I feel this is not a title, but I like your suggestion, so I will try to use the following: About the NamespaceManagement CR. WDYT?

> The namespace management feature in {gitops-shortname} lets tenants delegate control of their own namespaces to Argo CD instances without requiring cluster administrator intervention. By default, the feature is disabled for security reasons and must be explicitly enabled by a cluster administrator.

Reviewer: ```suggestion
The namespace management feature in {gitops-shortname} allows tenants delegate control of their own namespaces to Argo CD instances without requiring cluster administrator intervention.
```

When enabled, Argo CD administrators can configure namespace management using glob patterns to match one or more namespaces, such as `/\*` or `tenant-/\*`. Tenants then create a `NamespaceManagement` custom resource (CR) in their target namespace. The {gitops-title} Operator automatically provisions the required role-based access control (RBAC) resources and updates the Argo CD cluster secret, ensuring that the namespace is managed correctly.

> .Effects of `NamespaceManagement` CR

Reviewer: out of scope for now - cannot use block title in dita

Author: Fixed.
